pallets/parachain-staking/src/types.rs

```diff
@@ -34,10 +34,22 @@ use sp_std::{cmp::Ordering, prelude::*};
 
 pub struct CountedNominations<T: Config> {
     pub uncounted_stake: BalanceOf<T>,
-    pub rewardable_nominations: Vec<Bond<T::AccountId, BalanceOf<T>>>,
+    pub rewardable_nominations: BoundedVec<Bond<T::AccountId, BalanceOf<T>>, MaxNominations>,
 }
 
-#[derive(Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
+#[derive(Clone, Encode, Decode, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub struct MaxCloneableNominations;
```

Comment:
why do we need this? Couldn't we use a ConstU32 instead?

Reply:
to pass as a bound to `BoundedOrderedSet`